src/Simplex/Chat/Store.hs

```diff
@@ -595,6 +598,14 @@ deleteContact db user@User {userId} Contact {contactId, localDisplayName, active
   DB.execute db "DELETE FROM contacts WHERE user_id = ? AND contact_id = ?" (userId, contactId)
   forM_ customUserProfileId $ \profileId -> deleteUnusedIncognitoProfileById_ db user profileId
 
+deleteContactNoGroups :: DB.Connection -> User -> Contact -> IO ()
```

Comment:
how is it different from any contact deletion? what does "no groups" mean?

Reply:
doesn't check whether contact is used for any members - it's covered by Chat logic that contact has no groups - see deleteUnusedContact -> when (null ctGroups).

it's the same as old deleteContact function (save for last line), when we didn't allow to delete contacts that were in any groups